whotracksme/data/assets/trackerdb.sql

```diff
@@ -3498,6 +3536,26 @@ INSERT INTO "trackers" VALUES('sape.ru','Sape',6,NULL,'sape',NULL,NULL);
 INSERT INTO "trackers" VALUES('schibsted','Schibsted Media Group',8,'http://www.schibsted.com/','schibsted_asa',NULL,NULL);
 INSERT INTO "trackers" VALUES('shopifycdn.com','Shopify CDN',9,'https://www.shopify.com/','shopify','3064',NULL);
 INSERT INTO "trackers" VALUES('shopifycloud.com','Shopify Cloud',2,'https://www.shopify.com/','shopify',NULL,NULL);
+INSERT INTO "trackers" VALUES('orange_france','Orange France',8,'https://www.orange.fr/','orange_france',NULL,NULL);
+INSERT INTO "trackers" VALUES('dmcdn.net','Dailymotion',0,'https://www.dailymotion.com/','dailymotion',NULL,NULL);
```

Comment:
There is already a dailymotion entry in the trackers table. You can delete this entry.

Reply:
done